README.md

```diff
@@ -1,4 +1,32 @@
-# helm
-Helm Resources for RStudio Products
+# RStudio Helm charts
 
-This repository will eventually be made public, and it will contain public Helm charts for our products.
+## Usage
+
+1. Install [Helm](https://helm.sh). Please refer to Helm's [documentation](https://helm.sh/docs/) for more information on getting started.
+
+2. Add the RStudio Helm repo:
+
+   ```console
+   helm repo add rstudio https://helm.rstudio.com
+   ```
+
+3. View charts:
+
+   ```console
+   helm search repo rstudio
+   ```
+
+## Support
+
+We have a few outlets for getting support with our projects:
+
+- Bugs or feature requests should be opened in an [issue](https://github.com/rstudio/helm/issues/new/choose).
+- Professional support for our Helm charts can be obtained by emailing support@rstudio.com. Please note that email support is limited to our professional products customers.
```

Comment:
Yeah I think we should use similar wording to what we have in docker, since that was run by Amanda/etc.  At least until we've done more here.  Something similar to https://github.com/rstudio/rstudio-docker-products#rstudio-pro-products-docker-images